src/contentful/types.ts

```diff
@@ -0,0 +1,105 @@
+export type NebenanLocales = 'de-DE' | 'de' | 'en-US' | 'en' | 'fr-FR' | 'fr' | 'es-ES' | 'es' | 'it-IT' | 'it';
```

Comment:
should we really hardcode locales? do we transform them somewhere? otherwise I would just go with string there